src/sttn/data/restaurants.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import geopandas as gpd
+import numpy as np
+
+from sttn import network
+from .data_provider import DataProvider
+
+class RestaurantInspectionDataProvider(DataProvider):
+    """New York City Restaurant Inspections data provider, builds a network where nodes represent restaurants
+     and every edge represents an inspection where the origin and destination point to the node
+     where the inspection happened."""
+  
+    @staticmethod
+    def get_data(self, from_date, to_date) -> network.SpatioTemporalNetwork:
+        """
+        Retrieves New York City Restaurant Inspections data
+
+        Args:
+            from_date (str): A string with year and month and in the "YYYY-MM-DD" format.
+            Specifies the start date to get data from
+            
+            to_date (str): A string with year, month, and day in the "YYYY-MM-DD" format.
+            Specifies the end date to get data from
+                
+
+        Returns:
```

Comment:
Please add column-level documentation (see [this example](https://github.com/yuribogomolov/sttn/blob/main/src/sttn/data/nyc.py#L44))